editor/code_editor.cpp

```diff
@@ -1474,6 +1474,34 @@ void CodeTextEditor::goto_error() {
 	}
 }
 
+void CodeTextEditor::_update_text_editor_theme() {
+	text_editor->add_theme_color_override("background_color", EDITOR_GET("text_editor/highlighting/background_color"));
+	text_editor->add_theme_color_override("completion_background_color", EDITOR_GET("text_editor/highlighting/completion_background_color"));
+	text_editor->add_theme_color_override("completion_selected_color", EDITOR_GET("text_editor/highlighting/completion_selected_color"));
+	text_editor->add_theme_color_override("completion_existing_color", EDITOR_GET("text_editor/highlighting/completion_existing_color"));
+	text_editor->add_theme_color_override("completion_scroll_color", EDITOR_GET("text_editor/highlighting/completion_scroll_color"));
+	text_editor->add_theme_color_override("completion_font_color", EDITOR_GET("text_editor/highlighting/completion_font_color"));
+	text_editor->add_theme_color_override("font_color", EDITOR_GET("text_editor/highlighting/text_color"));
```

Comment:
It's worth noting that `font_color` and `font_color_selected` are the only theme items whose name differs from the matching editor setting (`text_color` and `text_selected_color`). Do we want to harmonize them?

Reply:
`font_color` is like our default font color property in a lot of controls 🤔 But `text_color` makes more sense for this particular editor setting. I'd keep it.

Though `font_color_selected` should be changed to `font_selected_color`.

Reply:
Nice catch should been `font_selected_color`.

Yeah `text` sounds better to me, though only for the editor, don't want to make `TextEdit` itself have a different naming scheme to the rest of the controls.